src/components/color-picker/gradient-select/GradientSelect.tsx

```diff
@@ -53,86 +54,60 @@ export interface GradientSelectProps {
   value: GradientScaleName;
   onChange: (value: GradientScaleName) => void;
 }
-
+const renderFilm: ItemRenderer<GradientScaleName> = (
```

Comment:
why "Film" ?

Reply:
Oh, sorry I forgot to rename it.
It's fromthe documentation example